install/install_for_macos_linux.sh

```diff
@@ -1,14 +1,21 @@
 #!/bin/bash
 
-if [ "$(uname)" == "Linux" ]; then
-  TIPI_URL="https://github.com/tipi-build/cli/releases/download/v0.0.25/tipi-v0.0.25-linux-x86_64.zip"
-fi
+VERSION=$TIPI_INSTALL_VERSION
 
-if [ "$(uname)" == "Darwin" ]; then
-  TIPI_URL="https://github.com/tipi-build/cli/releases/download/v0.0.25/tipi-v0.0.25-macOS.zip"
+if [  -z "$TIPI_INSTALL_VERSION" ]; then
+  if [ "$(uname)" == "Linux" ]; then
+    TIPI_URL="https://github.com/tipi-build/cli/releases/download/v0.0.25/tipi-v0.0.25-linux-x86_64.zip" 
+  elif [ "$(uname)" == "Darwin" ]; then
+    TIPI_URL="https://github.com/tipi-build/cli/releases/download/v0.0.25/tipi-v0.0.25-macOS.zip"
+  fi
+else 
+  if [ "$(uname)" == "Linux" ]; then
+    TIPI_URL="https://github.com/tipi-build/cli/releases/download/$VERSION/tipi-$VERSION-linux-x86_64.zip" 
+  elif [ "$(uname)" == "Darwin" ]; then
+    TIPI_URL="https://github.com/tipi-build/cli/releases/download/$VERSION/tipi-$VERSION-macOS.zip"
+  fi
```

Comment:
4 times the TIPI_URL? You can have a default variables in bash for version and this would greatly simplify updating the default version installed by the scripts.

See https://stackoverflow.com/a/2013589/271781

Reply:
Thanks for the info, it will simplify me other script in the future